WTWSMS/decisions/WtWSMS_realm_decisions.txt

```diff
@@ -413,11 +431,12 @@ decisions = {
 			OR = {
 				religion_group = christian
 				religion_group = pagan_group
+				has_romance_culture_trigger = yes
+				culture = roman
 			}
 			age = 16
 			prisoner = no
 			NOT = { trait = incapable }
-			has_romance_culture_trigger = yes
```

Comment:
Okay, I sort of understand this.

-but "UNITE THE ROMAN EMPIRE" requires that the Western and Eastern Roman Empires be under the same man. If both empires already exist, and both are held by one person, why shouldn't he unite them?

That a non-Romance character cannot restore a Roman Empire makes perfect sense, but if a non-Romance character assumes command of the existing Roman Empires, why can't he achieve the legacy?

I say this because you disabled the ability to create the Charlemagne Empire and the HRE. I tried to incorporate the HRE into the Mod, but you shut that down. Hence, I took this route.